### PR TITLE
Update AutoFeat documentation

### PR DESCRIPTION
**Pull request by Hiba-Alili**

### CODE REVIEW

**Review by dianajlailaty — APPROVED** [review]

File: src/docs/PML/PMLUserGuide.adoc [hunk 1]
```diff
@@ -2898,35 +2880,68 @@ NOTE: Your CSV file should be in a table format. See the example below.
...
 | String
 | `FILE_DELIMITER`
 | Defines a delimiter to use.
 | String (default=;)
 | `LIMIT_OUTPUT_VIEW`
-|  Specifies how many rows of the dataframe will be previewed in the browser to check each task results.
+|  Specifies how many rows of the dataframe will be previewed in the browser to check the encoding results.
 | Int (-1 means preview all the rows)
+| `NATIVE_SCHEDULER`
+| Name of the Native Scheduler node source to use when the workflow tasks must be deployed inside a cluster such as SLURM, LSF, etc.
```
Comment (mklkun) [inline src/docs/PML/PMLUserGuide.adoc:2920]: SLURM, LSF are not clusters but Schedulers.
must be deployed inside a cluster such as SLURM, LSF, etc. -> must be scheduled on a native scheduler such as SLURM, LSF, etc.

File: src/docs/PML/PMLUserGuide.adoc [hunk 2]
```diff
@@ -2898,35 +2880,68 @@ NOTE: Your CSV file should be in a table format. See the example below.
...
+| String (default=empty)
+| `NATIVE_SCHEDULER_PARAMS`
+|Parameters given to the native scheduler (SLURM, LSF, etc) while requesting a ProActive node used to deploy the workflow tasks.
```
Comment (mklkun) [inline src/docs/PML/PMLUserGuide.adoc:2923]: ProActive node used to deploy the workflow tasks. -> ProActive node used to execute the workflow tasks.

File: src/docs/PML/PMLUserGuide.adoc (hunk 1 above)
Comment (Hiba-Alili) [reply src/docs/PML/PMLUserGuide.adoc:2920]: SLURM, LSF, etc. refers to native schedulers. Indeed, it's a bit confusing. I'll reformulate the sentence. Thank you @mklkun  :)

**Review by mklkun — APPROVED** [review]
LGTM

File: src/docs/PML/PMLUserGuide.adoc [hunk 3]
```diff
@@ -2898,35 +2880,68 @@ NOTE: Your CSV file should be in a table format. See the example below.
...
+| String
+| `CONTAINER_PLATFORM`
+| Specifies the type of container platform to be used (no container, docker, singularity, or podman).
+| String (default=docker)
+| `CONTAINER_GPU_ENABLED`
+| If True, containers will run based on images containing libraries that are compatible with GPU.
+| Boolean (default=False)
+| `CONTAINER_IMAGE`
+| Specifies the name of the image that will be used to run the workflow tasks.
+| String (default=Empty).
```
Comment (ImenBizid) [inline src/docs/PML/PMLUserGuide.adoc:2942]: Please remove the AI workflows common variables. They are already described in this section https://trydev2.activeeon.com/doc/PML/PMLUserGuide.html#_ai_workflows_common_variables 
NATIVE_SCHEDULER
NATIVE_SCHEDULER_PARAMS
...
CONTAINER_IMAGE

**Review by ImenBizid — CHANGES_REQUESTED** [review]

**Review by ImenBizid — APPROVED** [review]

**Pull request merged**